Statistics/Distributions/IDistributionExtensions.cs

```diff
@@ -87,11 +87,25 @@ public static IDistribution FromXML(XElement ele)
                 {
                     switch (sa.type.Name){
                         case "double":
-                            double vald =  Convert.ToDouble(ele.Attribute(sa.Name).Value);
-                            pi.SetValue(dist,vald);
+                            double vald = 0.0;
+                            if(ele.Attribute(sa.Name).Value=="Infinity"){
+                                vald = double.PositiveInfinity;
+                            }else if(ele.Attribute(sa.Name).Value=="-Infinity"){
+                                vald = double.NegativeInfinity;
+                            }else{
+                                vald = Convert.ToDouble(ele.Attribute(sa.Name).Value);
```

Comment:
what is a vald?

Reply:
value lowercase d.  it is the value of the lowercase d double.

Reply:
is there a more descriptive option for naming?